projects/urls.py

```diff
@@ -5,9 +5,10 @@
 from django.conf.urls.static import static
 
 urlpatterns = [
-    path("admin/doc/", include("django.contrib.admindocs.urls")),
-    path("admin/", admin.site.urls),
+    # path("admin/doc/", include("django.contrib.admindocs.urls")), # Check error
+    # path("admin/", admin.site.urls), # Check error
```

Comment:
With the lines above I got a `LookupError: No installed app with label 'admin'.` and can't run the tests.